lib/sablon/content.rb

```diff
@@ -64,32 +65,44 @@ def append_to(paragraph, display_node)
       end
     end
 
+    # handles direct addition of WordML to the document template
     class WordML < Struct.new(:xml)
       include Sablon::Content
-      def self.id; :word_ml end
-      def self.wraps?(value) false end
+      def self.id
+        :word_ml
+      end
```

Comment:
One more thing, any particular reason you decided to reformat these inline methods? I think it does introduce more noise to this diff than necessary. Also, I like them for hardcoded definitions like the ones in these classes.